templates/content-certificate.php

```diff
@@ -54,6 +54,13 @@
 				</button>
 			<?php endif; ?>
 
+			<?php if ( $is_sharing_enabled ) : ?>
+			<span class="llms-button-secondary">
```

Comment:
I was envisioning a read-only text input in favor of a button that doesn't do anything when clicked.

Reply:
Hi, I thought the proposal mention "When sharing is enabled, a read-only text input should be output with the certificate's URL populated in it. "?

Reply:
Hi @imknight ,
yes so in place of this `<span>` that looks like a button, @thomasplevy was envisioning a read-only text input, as mentioned in the proposal.
Is that anything I'm missing?!

Reply:
oh got it, finally I understand, will update the code

Reply:
Have you had a look at our Installing for Development Docs: https://github.com/gocodebox/lifterlms/blob/trunk/docs/installing.md

Reply:
The CSS is probably less important than making sure this fundamentally works. Although now that I'm looking at this again we're going to need to add some UX to the field too...

I'm sure anyone who sees a field is also going to expect the following:

1) When you click the field it should highlight everything
2) There should be a "Copy" button next to the field

So we'll need to add some JS to this to power the UX

Reply:
I got it for 2 so that if a user clicks on the copy button, it will copy the permalink to the clipboard, but what is 1 about? highlight everything?

Reply:
Instead of having to double click (or single click and then ctrl/cmd + a) to highlight the whole link, it would be *simpler* for the end user for them to click once in the field and have the entire contents of the field automatically selected/highlighted.

Here's a quick gif of how google drive handles this (you maybe can't tell but I'm just clicking the field and it automatically highlights the whole contents)

![Peek 2020-10-20 09-25](https://user-images.githubusercontent.com/1290739/96615724-86cf1c00-12b6-11eb-852c-23aa3ba10036.gif)

I find that most apps that let you copy a link have this behavior automatically so if we *don't* do it we'll get support requests about how it doesn't work right, almost guaranteed.

Thanks!

Reply:
Also, just to test the concept, it can be done pretty simply with some JS like this:

```js
document.getELementById( 'test-input' ).addEventListener( 'focus', e => {
     e.target.select();
} );